src/ontology/mondo-edit.obo

```diff
@@ -552130,6 +552152,44 @@ xref: UMLS:C0005944 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source=
 is_a: MONDO:0005381 {source="NCIT:C97045"} ! bone disorder
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7006" xsd:anyURI
 
+[Term]
+id: MONDO:0800496
+name: childhood-onset self-limited focal epilepsy syndrome
+def: "A group of conditions characterized by age-dependent occurrence in otherwise normal children. Cognition and neurological evaluation are typically normal. Remission occurs in almost all patients by puberty. Presumed genetic factors have an important role. Seizure semiology and electroencephalographic (EEG) features are specific for each of the syndromes included in this group." [PMID:35503717]
+synonym: "childhood-onset SeLFE" EXACT [https://www.epilepsydiagnosis.org/syndrome/epilepsy-syndrome-groupoverview.html]
+is_a: MONDO:0020072 {source="https://www.epilepsydiagnosis.org/syndrome/epilepsy-syndrome-groupoverview.html"} ! childhood-onset epilepsy syndrome
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8490" xsd:anyURI
+
+[Term]
+id: MONDO:0800498
+name: childhood-onset genetic generalized epilepsy syndrome
+def: "A genetic generalized epilepsy that has an onset during childhood." [https://orcid.org/0000-0001-5208-3432]
+is_a: MONDO:0020072 {source="https://orcid.org/0000-0001-5208-3432"} ! childhood-onset epilepsy syndrome
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8490" xsd:anyURI
+
+[Term]
+id: MONDO:0800499
+name: childhood-onset idiopathic generalized epilepsy syndrome
+def: "An idiopathic generalized epilepsy that has an onset during childhood." [https://orcid.org/0000-0001-5208-3432]
+is_a: MONDO:0020072 {source="https://orcid.org/0000-0001-5208-3432"} ! childhood-onset epilepsy syndrome
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8490" xsd:anyURI
+
+[Term]
+id: MONDO:0800500
+name: childhood-onset epilepsy syndrome with developmental and/or epileptic encephalopathy
+def: "An epilepsy syndrome where the onset of the condition includes manifestations of cognitive, neurological, or psychiatric impairment, stagnation, or regression, due directly to the underlying etiology. In contrast, an epileptic encephalopathy (EE) is present when the encephalopathy is caused by the epileptic activity. The term developmental and epileptic encephalopathy (DEE) is used when both factors contribute to the patient’s condition." [https://www.epilepsydiagnosis.org/encephalopathy.html]
```

Comment:
I know that the parent indicates that it has a childhood onset, but I think it would still be useful to add it in this definition.

Reply:
I updated the def